### ISSUE TITLE
Searching CYP11B1

### ISSUE THREAD

**Issue by lohnomac** [biocaddie/prototype_issues#156]
There is a suggestion "did you mean CYP12B1". Then when you type CYP12B1 it says there are no results? Why was it suggested in the first place?

**Comment by ianfore** [comment]
I've noticed this with other queries - it gave me a "did you mean" and there were no results for it. 

I'm guessing that the autocompletion/"did you mean" is driven by what's on the ontology server rather than what's in the index.

**Comment by ianfore** [comment]
Here's a "did you mean" example I came across yesterday
Entered "Non-Hodgkins GWAS" as my search i.e. what seems to me a quite reasonable thing to type if I was looking for Genome Wide Association Studies on Non-Hodgkin's Lymphoma.
Datamed asked "Did you mean non hodgkin's gnas ?"
This seems rather random and gives no impression that the tool "understands" what I am trying to look for.

If I search for "Non-Hodgkin's GWAS" i.e. using the apostrophe
Datamed asks "Did you mean non hodgkin 39 s gnas ?"
In other words it has parsed out 39 (the ASCII code for an apostrophe) as being a meaningful part of the query.

The "did you mean" functionality needs serious attention.

**Comment by ianfore** [comment]
For comparison. In Google, if I mistakenly type "non hodkins gwas" it
a) Takes matters into its own hands and actually searches for "non hodgkin's gwas"
b) Tells me it did so
c) Gives me highly relevant results

Bing also does a) and b) in a slightly different way
"No results found for non hodkins gwas.
Showing results for non hodgkins gwas."
It shows some relevant results but more false positives ( less precise).

Tools like Google and Bing set the bar for what users expect for "Did you mean".
